virtual_rainforest/models/litter/constants.py

```diff
@@ -43,6 +61,25 @@ class LitterConsts:
     Value is taken from :cite:t:`kirschbaum_modelling_2002`.
     """
 
+    litter_decay_constant_woody: float = 1.0 / 150.0
+    """Decay constant for the woody litter pool [day^-1].
+
```

Comment:
Why did you chose to provide the values as fractions?

Reply:
This was how they were given in the paper I took them from, so I wanted to use the same to preserve numerical accuracy